llama-index-core/llama_index/core/schema.py

```diff
@@ -25,7 +25,7 @@
     Sequence,
     Union,
 )
-
+from pydantic import SerializationInfo, SerializerFunctionWrapHandler
```

Comment:
nit: `from llama_index.core.bridge.pydantic import ...` -- if its not already exported there, we can add it